astropy/stats/funcs.py

```diff
@@ -835,7 +885,8 @@ def mad_std(data, axis=None, func=None):
     """
 
     # NOTE: 1. / scipy.stats.norm.ppf(0.75) = 1.482602218505602
```

Comment:
@mhvk I did describe this factor in the docstring:
http://docs.astropy.org/en/stable/api/astropy.stats.mad_std.html#astropy.stats.mad_std

The factor is the normal inverse cumulative distribution function evaluated at P=3/4.  It's exactly `1. / scipy.stats.norm.ppf(0.75)`.